dashboard/app/models/user.rb

```diff
@@ -1912,9 +1917,9 @@ def section_scripts
     all_scripts
   end
 
-  # return the id of the section the user most recently created.
+  # return the id of the most-recently-created section the user instructs.
   def last_section_id
-    teacher? ? sections.where(hidden: false).last&.id : nil
+    teacher? ? sections_instructed.where(hidden: false).last&.id : nil
```

Comment:
If we are redefining sections, why do we need this?

Reply:
We don't. It should work the same either way. I just started with doing the whack-a-mole, didn't change this one back.

Reply:
I think if we are going to commit to this we should remove all cases where we are calling sections_instructed instead of just sections.

We could do this in a FLUP PR next week and I can create a jira task, but I really don't want the added tech debt of half migrating to `sections_instructors` and then redefining `sections`

Reply:
I could go either way on which synonym to prefer; we have `sections_as_student` already to define the participant->section relationship and using `sections_instructed` when possible would be a nice way to avoid ambiguity. Either way if we want to get this into staging before the bug bash I'd prefer not touching it to avoid the risk of a flaky drone run derailing things.

Reply:
Tracking in [TEACH-734](https://codedotorg.atlassian.net/browse/TEACH-734)

Reply:
does making this change effect how we would calculate a "most recent" section? Previously, we would have said the teacher's last section was the last owned they _owned_; now it would be last section they **instructed**?

Reply:
For example: if we are counting how many sections a teacher taught (owned) vs instructed.

Reply:
I don't know that this specific PR will affect us. The whole notion of having co-teachers will affect us but I believe that can be handled by the new section_instructor table in this PR [https://github.com/code-dot-org/code-dot-org/pull/53980/files].
@jordan-springer To answer your question of how we will use owned/instructed, we used to link teachers to sections by 'owned', but now we will need to do it by 'instructed'. This will also impact how we join students all the way to schools (e.g. what happens if teachers of a co-taught section have different schools?).

Reply:
I think using sections instructed rather than owned would likely make sense for most data analysis purposes. All that section "ownership" means within the product is that user hit the button to create the section, and they can't be removed as an instructor. Beyond that, the second (and subsequent) instructors are intended to have all the same abilities as the first. Any queries that continue to look only at the section owner will be capturing one of the instructors, and missing the rest.